src/styles/index.scss

```
@@ -1,3 +1,89 @@
-body {
+$product-font-color: #060b35;
+
+@import 'stars';
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.product-cards-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+    background-size: contain;
```

`img` isn't a background
```suggestion
    width: 160px;
    height: 134px;
    margin: 32px 19px 0;
```